package.json

```diff
@@ -8,19 +8,19 @@
   "main": "typechain-types/index.js",
   "types": "typechain-types/index.d.ts",
   "files": [
-    "proxies.*",
+    "network.*",
     "contracts",
     "typechain-types/**/*.js?(.map)",
     "typechain-types/**/*.d.ts"
   ],
   "exports": {
     ".": "./typechain-types/index.js",
-    "./proxies.js": "./proxies.js",
+    "./network.js": "./network.js",
```

Comment:
Yeh I guess this means a major version bump? Although, is anyone other than _us_ using this? It is totally "undocumented".

Reply:
That's a good point.  Since we are the only ones using this, we could get away without a major bump.  But... We have to make sure to publish this and an update to the v3 sdk at the same time.  Anyone who does `npm install @tableland/sdk` while the sdk is still trying to import "@tableland/evm/proxies.js" would get a broken setup.

Reply:
may as well major bump it so be safe. i'll create some issues on sdk and cli to handle the new version.

Reply:
(plus we have the changes in SQLHelpers that demand a major bump)